libr/bin/format/elf/elf.c

```diff
@@ -2102,6 +2102,28 @@ char* Elf_(r_bin_elf_get_arch)(ELFOBJ *bin) {
 	}
 }
 
+char* Elf_(r_bin_elf_get_head_flag)(ELFOBJ *bin){
+	if (bin->phdr && bin->ehdr.e_machine == EM_MIPS) {
+		const ut32 mipsType = bin->ehdr.e_flags & EF_MIPS_ARCH;
```

Comment:
spaces and tabs

Reply:
Thank you for your patience. I will check the code format carefully next time.